control/statefbk.py

```diff
@@ -252,13 +253,13 @@ def acker(A, B, poles):
 
     # Place the poles using Ackermann's method
     n = np.size(p)
-    pmat = p[n-1]*a**0
+    pmat = p[n-1] * np.linalg.matrix_power(a, 0)
```

Comment:
`np.eye(a.shape)` ?  I suppose the `matrix_power` call sets the stage for the upcoming for-loop.